app/policies/casa_case_policy.rb

```diff
@@ -40,6 +40,7 @@ def assign_volunteers?
 
   def permitted_attributes
     common_attrs = [
+      :court_date,
```

Comment:
Since volunteers can't update the court date, let's add a more specific case for `CasaAdmin` and `Supervisor` like so:

```
    case @user
    when CasaAdmin
      common_attrs.concat(%i[case_number birth_month_year_youth])
    when CasaAdmin, Supervisor
      common_attrs.concat(%i[court_date])
    else
      common_attrs
    end
```

Reply:
Done. I made this two separate cases as the admin seemed to only get the first set it matched.

Reply:
Whoops, my bad, thanks for catching that!